src/components/dialogs/active-power-control/active-power-control-form.tsx

```diff
@@ -54,9 +61,7 @@ const FrequencyRegulation = ({ isEquipmentModification, previousValues }) => {
             name={DROOP}
             label={'Droop'}
             adornment={percentageTextField}
-            previousValue={
-                !isNaN(previousValues?.activePowerControl?.droop) ? previousValues?.activePowerControl?.droop : null
-            }
+            previousValue={previousValues?.droop ?? undefined}
```

Comment:
```suggestion
            previousValue={Number.isNaN(previousValues?.droop) ? undefined : previousValues?.droop ?? undefined}
```
I suggest keep the same intention of the existing implementation. In fact NaN is also in 'number' type and i think the backend may be return a NaN in some case